client/src/views/client-area/ClientAreaPage.vue

```diff
@@ -197,7 +200,7 @@ onMounted(async () => {
       currentPage.value = ClientAreaPages.Contracts;
       const statusResp = await BmsAccessInstance.get().getCustomOrderStatus(currentOrganization.value);
       if (!statusResp.isError && statusResp.data && statusResp.data.type === DataType.CustomOrderStatus) {
-        if (statusResp.data.status === CustomOrderStatus.NothingLinked) {
+        if (statusResp.data.status !== CustomOrderStatus.ContractEnded) {
```

Comment:
Now it's going to be displayed all the time except for ContractEnded, which means the Contracts page will never be shown.